schema.json

```diff
@@ -184,12 +219,20 @@
             "type": "string"
           }
         },
+        "required": [
+          "institution",
+          "area",
+          "studyType",
+          "startDate",
+          "endDate"
```

Comment:
Why is endDate required? Didn't we think that endDate not being provided means current?

Reply:
Sorry I will fix it.